craft_parts/parts.py

```diff
@@ -68,6 +68,9 @@ class PartSpec(BaseModel):
     build_attributes: list[str] = []
     organize_files: dict[str, str] = Field(default_factory=dict, alias="organize")
     overlay_files: list[str] = Field(default_factory=lambda: ["*"], alias="overlay")
+    backstage_files: list[RelativePathStr] = Field(
```

Comment:
hmm do we need this? I thought we weren't going to expose the backstage to the users
it's like disney, looking behind the scenes breaks the magic

Reply:
We need this for the migrations to work.

Reply:
why? You're passing "*" here:

```
            backstage_files, backstage_dirs = filesets.migratable_filesets(
                Fileset(["(default)/*"], name="backstage"),
                str(self._part.part_export_dir),
                "default",
            )
```

This is the model, not the state